### ISSUE TITLE
Dismiss Button for Open Source Page Is Read Incorrectly When Using TalkBack

### ISSUE THREAD

**Issue by JB184351** [woocommerce/woocommerce-android#7406]
**Describe the bug**
Similar to the dismiss issue on the feedback screen the symbol for dismissing is an X and TalkBack reads this back as Navigate Up button, perhaps the symbol could be changed to a back button like in other parts of the app?
**To Reproduce**

1. Go to Menu
2. Click on Settings
3. Scroll down to `Open Source Licenses`
4. Select Open Source Libraries
5. Activate TalkBack
6. Select the Dismiss Button
7. Notice it reads `Navigate Up button`

**Screenshots**
<img src="https://user-images.githubusercontent.com/36495451/190718682-7f9d6f76-f45d-47e8-9584-e8fe424d1111.jpg" width="300"/>

**Expected behavior**
This should Dismiss button instead of Navigate back button

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: Samsung Galaxy A51
- Android version: 12
- WooCommerce Android version: 10.1

**Comment by JorgeMucientes** [comment]
To be honest, I don't think we'll ever tackle this. So, for now, I'm closing it.
